test/functional/admin/legacy_world_location_news_controller_test.rb

```diff
@@ -0,0 +1,126 @@
+require "test_helper"
+
+class Admin::WorldLocationNewsControllerTest < ActionController::TestCase
+  setup do
+    login_as :writer
```

Comment:
we need to add the "Preview design system" permission to the user for the non-legacy tests

```
test/functional/admin/lworld_location_news_controller_test.rb:5

login_as_preview_design_system_user(:writer)
```